MarkBind/markbind
# Fix extra newline in convert feature

**What is the purpose of this pull request?**




Fixes #1726 
Closes #1727 

**Overview of changes:**

This was caused by the extra newline created at the end of buildSiteNav() in the core/site dir. I simply added a trimEnd() to fix the issue.

**Anything you'd like to highlight / discuss:**


**Testing instructions:**

Run `markbind init -convert` on a test repo and see that there is no more new line before the closing </site-nav> tag.
When running tests, the Home.html in `test_navigation_convert` should not contain any unnecessary spaces in the navigation component. 



**Proposed commit message: (wrap lines at 72 characters)**

Fixed convert command generating extra newline



---

**Checklist:** :ballot_box_with_check:
